gpii/node_modules/preferencesServer/src/preferencesService.js

```diff
@@ -73,6 +108,41 @@ var fluid = fluid || require("infusion");
                 listener: "gpii.preferencesServer.preferencesService.updateGpiiKey",
                 args: ["{dataStore}.updateGpiiKey", "{arguments}.0"],
                 namespace: "updateGpiiKey"
+            }],
+            onCloudSafeCredCreate: [{
+                func: "{dataStore}.findById",
+                args: ["{arguments}.0"],
+                namespace: "findById"
+            },
+            {
+                func: "{expressUserUtils}.createNewUser",
```

Comment:
Curious what would happen if the given user name already exists. Can you add a test case for it too? Thanks.

Reply:
The underlying API will return an error, but it's good to test the handling and relaying of that error in this context.

Reply:
I've added a test for this, and also added a message from the prefs server adding more detail to the couch conflict error raised by gpii-express-user.